ERC20ForSPL/scripts/deployERC20ForSPLMintableThruFactory.js

```diff
@@ -4,31 +4,33 @@
 // You can also run a script with `npx hardhat run <script>`. If you do that, Hardhat
 // will compile your contracts, add the Hardhat Runtime Environment's members to the
 // global scope, and execute the script.
-const { ethers, upgrades } = require('hardhat');
-const { expect } = require('chai');
+const { ethers } = require("hardhat");
 
 async function main() {
-    [owner] = ethers.getSigners();
+  const ERC20ForSPLFactoryAddress = "";
```

Comment:
I would declare this constant outside the function and name it upper-snake case `ERC20ForSPL_FACTORY_ADDRESS` or just `FACTORY_ADDRESS`. IMO all constants should be like that in the beginning of a file so people don't forget to assign value to them

Reply:
Makes sense, @0xlucian !